src/bitmessagemain.py

```diff
@@ -1,4 +1,8 @@
 #!/usr/bin/python2.7
+"""
+src/bitmessagemain.py
+=================================
```

Comment:
please remove and write a docstring instead

Reply:
Okay. Thank you

Reply:
What should the docstring for bitmessagemain ?

Reply:
Can you please tell me the docstring for bitmessagemain file.

Reply:
"""The PyBitmessage startup script""
for example

Reply:
Notice, https://github.com/Bitmessage/PyBitmessage/blob/v0.6/docs/conf.py#L227 - this module is excluded from autodoc generation.

Reply:
Yes, i noticed that. But why it's excluded from autodoc generation?

Reply:
because it breaks doc build in current state

Reply:
https://pybitmessage-test.readthedocs.io/en/latest/autodoc/pybitmessage.bitmessagemain.html

Reply:
Okay. Thank you so much